fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/response/DruidResponseSerializer.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.model.response;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+
+/**
+ * Serializes {@link DruidResponse} into a json equivalent to Druid's actual responses.
+ */
+public class DruidResponseSerializer extends JsonSerializer {
+
+    @Override
+    public void serialize(Object o, JsonGenerator jsonGenerator, SerializerProvider serializerProvider)
+            throws IOException {
```

Comment:
I think JSON parsing exception should be handled inside the method, which means we should throw in a `try-catch`; you don't want to pass this method-logic to API callers.